light-blockchain/src/push.rs

```diff
@@ -105,21 +100,27 @@ impl LightBlockchain {
         let chain_info = ChainInfo::from_block(block, &prev_info);
 
         // More chain ordering.
-        match chain_order {
+        let result = match chain_order {
             ChainOrdering::Extend => {
                 return LightBlockchain::extend(this, chain_info, prev_info);
             }
             ChainOrdering::Superior => {
                 return LightBlockchain::rebranch(this, chain_info, prev_info);
             }
-            ChainOrdering::Inferior => unreachable!(),
-            ChainOrdering::Unknown => {}
-        }
+            ChainOrdering::Inferior => {
+                log::debug!(block = %chain_info.head, "Storing block - on inferior chain");
```

Comment:
Seems strange that we're logging this here and not in the other cases. Also I'd change the message to something like:
`log::debug!(block = %chain_info.head, "Ignore block - on inferior chain")`

Reply:
Im following the same pattern that is used in the regular blockchain.
https://github.com/nimiq/core-rs-albatross/blob/albatross/blockchain/src/blockchain/push.rs#L102

Also we are not ignoring the block, we are storing the block as part of the chain info in line 121. 
That was actually the bug that was fixed.

Reply:
We don't log in the other cases, because those cases are logged by the extend/rebranch functions and not from the caller side